samples/net/aws_iot/boards/nrf7002dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Configuration file for nRF7002 DK
+# This file is merged with prj.conf in the application folder, and options
+# set here will take precedence if they are present in both files.
+
+# General
+CONFIG_POSIX_CLOCK=y
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
+CONFIG_LOG_BUFFER_SIZE=3072
+CONFIG_HEAP_MEM_POOL_SIZE=153600
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_HW_ID_LIBRARY_SOURCE_NET_MAC=y
+
+# Set newlib C to prevent build error complaining that string.h is not supported
+CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
+
+# Wi-Fi
+CONFIG_WIFI=y
+CONFIG_WIFI_NRF700X=y
+CONFIG_WIFI_LOG_LEVEL_ERR=y
```

Comment:
We use this option all over the place, also in the Azure sample. Just curious if we actually need it. If it was originally introduced to save flash, I think we can drop it

Reply:
There are a couple of log-infs in the wifi driver. Ill make them debug and remove the option from the sample.

[00:00:05.696,899] <inf> wifi_nrf: wifi_nrf_wpa_supp_authenticate:Authentication request sent successfully
[00:00:05.966,674] <inf> wifi_nrf: wifi_nrf_wpa_supp_associate: Association request sent successfully